### ISSUE TITLE
Zapddit is looks like Dev.to tags not reddit

### ISSUE THREAD

**Issue by pj8912** [vivganes/zapddit#32]
zapddit looks exactly like [http://dev.to](https://t.co/TpntYNGSk5) follow tags than reddit. 
👇👇

### Tags:
![Screenshot from 2023-06-15 21-17-57](https://github.com/vivganes/zapddit/assets/59218902/29ff3cec-1694-4be9-9e78-ae34279c7896)

### Tag with moderators:
![Screenshot from 2023-06-15 21-18-09](https://github.com/vivganes/zapddit/assets/59218902/132561ba-5fb3-448c-a484-b826d87b3e13)

### Followed Tags:
![image](https://github.com/vivganes/zapddit/assets/59218902/d66ed375-13db-4dc3-a427-618bb4936d87)

Zapddit looks nothing like reddit. Why the comparison to reddit??

**Comment by vivganes** [comment]
Interesting question @pj8912 

You are being too generous 😆  I wish zapddit's UI is half beautiful as dev.to's UI, which clearly pops 🤩 

Now, to answer your question,

**Short answer:** [An extract](https://github.com/vivganes/zapddit#reddit-vs-zapddit) from the readme

**A longer answer:**
1. I created zapddit as a weekend project in May 2023.  At that time, I didn't expect this would garner any attention at all.
2. I am a redditor and am used to following 'topics/communities' instead of people. I wanted a client that I could use for nostr.  So, I built zapddit using [NDK](https://github.com/nostr-dev-kit/ndk) by @pablof7z
3. zapddit is a **nostr-client first**, then  a reddit-style client, in that particular order
4. Similar to upvote and downvote, I built the upzap and downzap features, in line with other nostr clients way of zapping notes.
5. Since there is no equivalent of communities in nostr, I had to settle for hashtags so that the following of topics can happen, even without communities.
6. Finally, the thought process behind the reddit comparison is [here](https://github.com/vivganes/zapddit#reddit-vs-zapddit)

In short, zapddit is for those who want to use nostr, but miss the reddit-style browsing of topics, upvoting, downvoting, etc.

Like I said in a [different answer](https://github.com/vivganes/zapddit/issues/18#issuecomment-1589329838), zapddit is not meant to be a reddit replacement.  It probably will never become one.

**Comment by pj8912** [comment]
Yeah, [Bija](https://github.com/BrightonBTC/bija) already have subscribe to topic feature. Which is just one of the feature of the client.
